syft/pkg/cataloger/debian/cataloger_test.go

```diff
@@ -25,15 +25,16 @@ func TestDpkgCataloger(t *testing.T) {
 					Version: "1.1.8-3.6",
 					FoundBy: "dpkg-db-cataloger",
 					Licenses: pkg.NewLicenseSet(
-						pkg.NewLicenseFromLocations("GPL-1", file.NewVirtualLocation("/usr/share/doc/libpam-runtime/copyright", "/usr/share/doc/libpam-runtime/copyright")),
-						pkg.NewLicenseFromLocations("GPL-2", file.NewVirtualLocation("/usr/share/doc/libpam-runtime/copyright", "/usr/share/doc/libpam-runtime/copyright")),
-						pkg.NewLicenseFromLocations("LGPL-2.1", file.NewVirtualLocation("/usr/share/doc/libpam-runtime/copyright", "/usr/share/doc/libpam-runtime/copyright")),
+						pkg.NewLicenseFromLocations("GPL-1", file.NewLocation("/usr/share/doc/libpam-runtime/copyright")),
```

Comment:
Can you help me understand why this stopped being a virtual location and became a regular location? Is this change visible to users in the eventual SBOM?

Reply:
This was cleanup -- since I was in this file and making new Locations, I noticed that all params to the original call were the same, which is the same as using the NewLocation constructor. So I made the remaining parts of the file that were using this function needlessly as consistent as the new code I was adding.